crates/sui-framework/Cargo.toml

```diff
@@ -15,6 +15,7 @@ smallvec = "1.9.0"
 num_enum = "0.5.7"
 once_cell = "1.11.0"
 sha3 = "0.10.1"
+curve25519-dalek-ng = "4.1.1"
```

Comment:
Wondering if we need any external crypto lib dep here. Ideally we should only import our Narwhal crypto deps. 

Adding it as dev-dep for testing only would also be an option, but I'd prefer the Narwhal crypto approach.

Reply:
Yep, I agree - but could we do this in a separate PR? Narwhal bulletproofs is already merged and this is just importing this for the u64 -> bytes converter. It's literally just being used as a BigInt here.

Reply:
#4122